Cakes-Co/style.css

```diff
@@ -1 +1,153 @@
-/*  Add your styling here */
+* {
+  margin: 0px;
+  padding: 0px;
+}
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0px;
+  padding: 0px;
+}
+img {
+  width: 100%;
+  height: 100%;
+}
+:root {
+  --bar-width: 30px;
+  --bar-height: 5px;
+  --gap: 30px;
+}
+
+.logo-img {
+  width: 100px;
+  height: 100px;
+  align-self: center;
+}
+header {
+  background-color: blanchedalmond;
+  width: auto;
+  height: 140px;
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+  padding: 30px;
+}
+.inner-header {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+.motto-p {
+  text-align: right;
+  max-width: 20ch;
+}
+
+.ham-label {
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+  z-index: 3;
+}
+
+.ham-label input,
+.ham-label::before,
+.ham-label::after {
+  content: "";
+  width: var(--bar-width);
+  height: var(--bar-height);
+  background-color: black;
+}
+
+.ham-label input {
+  appearance: none;
+  outline: none;
+  pointer-events: none;
+}
+
+.menu-aside {
+  position: absolute;
+  background-color: grey;
+  width: 220px;
+  height: 400px;
+  top: 0px;
+  right: 0px;
+  z-index: 2;
+  display: none;
+}
+
+.ham-label:has(input:checked) + .menu-aside {
+  display: block;
+}
+
+nav {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+main {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(3, auto);
+  gap: var(--gap);
+}
+
+.p-sec {
+  grid-column: 1/-1;
+  grid-row: 1/2;
+}
+.hero-sec {
+  grid-column: 1/-1;
+  grid-row: 2/3;
+}
+.img-sec {
+  grid-column: 1/-1;
+  grid-row: 3/-1;
+
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(2, auto);
+  gap: var(--gap);
+}
+
+@media screen and (min-width: 640px) {
```

Comment:
This is good - it's responsive and mobile first as requested  by the intructions